config/default.go

```diff
@@ -54,6 +54,7 @@ GlobalQueue = 1024
 	MaxConns = 200
 
 [Etherman]
+IsRollup = true
```

Comment:
I see `Etherman`, `Synchronizer` and `SequenceSender` have a new config parameter called `IsRollup`, do you think we could centralize and share this config in the root config the same way as `IsTrustedSequencer`?

This way we would have a single point to define this and this would be replicated to all the components running this configuration.

Reply:
The problem of doing such a thing is that then all the constructors will need a new parameter `isRollup` and this will need to be stored on the root of each struct (or elsewhere, but not in the config). I still thing that it's a good idea to apply the change since this will make the config file better which is already complex enough, and it's what the users will see at the end of the day

Reply:
you can maintain the config structure you already created, just create a new field in the root config with the `mapstrucutre` tag, remove all the `mapstructure` tags from the internal configs and, do the set mapping in the `run.go`, here is an example:

state configs inherited from other components configs:
https://github.com/0xPolygonHermez/zkevm-node/blob/496d4a13f76b69fbb356a24019d336cccbf26d33/cmd/run.go#L482-L492

RPC configs with `mapstructure` tags to read data from config:
https://github.com/0xPolygonHermez/zkevm-node/blob/496d4a13f76b69fbb356a24019d336cccbf26d33/jsonrpc/config.go#L50-L56

State configs without `mapstrucutre` tags to be set via `run.go`:
https://github.com/0xPolygonHermez/zkevm-node/blob/496d4a13f76b69fbb356a24019d336cccbf26d33/state/config.go#L37-L43